gateway/src/apicast/policy/maintenance_mode/policy.lua

```diff
@@ -0,0 +1,37 @@
+-- This is a simple policy. It allows you reject incoming requests with a
+-- specified status code and message.
+-- It's useful for maintance periods or to temporarily block an API
+
+local _M = require('apicast.policy').new('Maintenance mode')
+
+local tonumber = tonumber
+local new = _M.new
+
+
+function _M.new(configuration)
+  local policy = new(configuration)
```

Comment:
There's no need to pass `configuration` as a param here.

Reply:
This made me realize that we are doing the same in the echo policy. We'll need to fix that.

Reply:
@davidor I think we might want to keep this as this is the signature of the initializer. That the underlying policy does not do anything with it is an implementation detail that can change in the future. What if the policy would actually store it in `self.configuration` ? We would have to change all policies to pass it again.

Reply:
@mikz I see what you mean, but `new()` here is calling `function policy.new()` in `policy.lua` which does not accept any parameters. Maybe we should add it then to make it less confusing?

Reply:
@davidor Ah damn. I actually followed the echo as my baseline. Are you both happy for me to keep this as the signature or change it for now?